fontforge/tottfvar.c

```diff
@@ -572,26 +572,12 @@ return;
 	}
 	/* Now output the corresponding deltas for those points */
 	for ( j=0; j<pcnt; ) {
-	    if ( deltas[i][j]>0x7f || deltas[i][j]<0x80 ) {
```

Comment:
These changes obviously don't affect the behavior but they make me nervous. But this is a MultiMaster thing and while that code may be revived at some point for VF support it's effectively dead now.

Reply:
I tried to make some sense of this but it appears whole-sale in 6243f733fb327c99b91987be95a70712e9b69a3c with the initial addition of this file in 2004 and hasn't changed since, so if it behaved correctly when tested then, then clearly this was meant to be a dead branch?

Reply:
It's hard to judge because the multi-master code was almost never used. The implementation is at best incomplete.